packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -2490,6 +2497,16 @@ export class ProjectService extends BaseService {
             throw new ForbiddenError();
         }
 
+        const cachekey = `${projectUuid}-${table}-${initialFieldId}-${search}-${JSON.stringify(
+            filters,
+        )}`;
+        if (
```

Comment:
Should we add an environment variable to control this feature ? 
I don't want to use `feature flags` in the backend.